tools/packager/afu.py

```diff
@@ -32,13 +32,8 @@
 import zipfile
 from metadata import metadata
 from gbs import GBS, GBS_EXT
-
-try:
-    import jsonschema
-except ImportError:
-    print("afu.py requires the jsonschema package.  Please install it.")
-    print("  https://pypi.python.org/pypi/jsonschema" + os.linesep)
-    raise
+from jsonschema import validators
```

Comment:
`validators` and `exceptions` are terribly generic names to be pulling into global namespace unqualified...